ui/scripts/inlineHtml/inlineScripts.js

```diff
@@ -880,18 +901,18 @@ const updateSkillDisciplineExpertises = function (section) {
       id => `repeating_${section}_${id}_skilldisciplineexpertise`);
     getAttrs(disciplineinfos.concat(isdisciplines).concat(expertises)
       .concat(disciplineexpertises),
-    function(attributes) {
+    function (attributes) {
       let discipline_expertise = 0;
       let update = {};
       for (let i = 0; i < ids.length; ++i) {
         if (attributes[disciplineinfos[i]] === 'D') {
           discipline_expertise = attributes[expertises[i]];
-          if (attributes[isdisciplines[i]] != 1) {
-            update[isdisciplines[i]] = 1;
+          if (attributes[isdisciplines[i]] !== '1') {
```

Comment:
This was one case where the value was ambiguous.
All attributes are strings, so this was not quite right.

Reply:
Seems like these if block is not truly needed. Maybe it sort of ensures we don't set attributes if they are already set.
We could just update it regardless.

In addition, this property is only used to show or hide UI elements in CSS
Instead, we could use the discipline value directly, since only `D` is relevant.
The CSS checking for `value="1"`, could instead check the disciplineinfo field for `value="D"`